lisp/mastodon-tl.el

```diff
@@ -163,7 +245,10 @@ Return value from boosted content if available."
              (mastodon-tl--byline-author toot)
              (mastodon-tl--byline-boosted toot)
              " "
-             (format-time-string mastodon-toot-timestamp-format (date-to-time timestamp))
+             (propertize
+              (format-time-string mastodon-toot-timestamp-format parsed-time)
+              'timestamp parsed-time
+              'display (mastodon-tl--relative-time-description parsed-time))
```

Comment:
Could we use the non relative timestamp here when `mastodon-tl--enable-relative-timestamps` is nil?

``` emacs-lisp
    'display (if mastodon-tl--enable-relative-timestamps
                           (mastodon-tl--relative-time-description parsed-time)
                         parsed-time)
```